### PR TITLE
[Enhancement](spark load)Support spark time out config

### PR DESCRIPTION
**Pull request by liujinhui1994**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/test/java/org/apache/doris/load/loadv2/SparkLauncherMonitorTest.java [hunk 1]
```diff
@@ -57,7 +58,7 @@ public void testLogMonitorNormal() {
         try {
             Process process = Runtime.getRuntime().exec(cmd);
             handle = new SparkLoadAppHandle(process);
-            SparkLauncherMonitor.LogMonitor logMonitor = SparkLauncherMonitor.createLogMonitor(handle);
+            SparkLauncherMonitor.LogMonitor logMonitor = SparkLauncherMonitor.createLogMonitor(handle, new HashedMap());
```
Comment (lide-reed) [inline fe/fe-core/src/test/java/org/apache/doris/load/loadv2/SparkLauncherMonitorTest.java:61]: Can you create a  hashmap which include SUBMIT_TIMEOUT_KEY info?

**Review by lide-reed — APPROVED** [review]
LGTM

**Review by lide-reed — DISMISSED** [review]
LGTM

**Comment by lide-reed** [comment]
run buildall

**Review by lide-reed — DISMISSED** [review]
LGTM

**Comment by lide-reed** [comment]
run buildall

**Comment by liujinhui1994** [comment]
run feut

**Comment by liujinhui1994** [comment]
run buildall

**Comment by Yulei-Yang** [comment]
run buildall

**Comment by liujinhui1994** [comment]
run buildall

**Review by lide-reed — APPROVED** [review]
LGTM

**Comment by liujinhui1994** [comment]
run p0

**Pull request merged**